ansible_collections/arista/avd/plugins/filter/snmp_hash.py

```diff
@@ -8,6 +8,51 @@
 
 from ansible.errors import AnsibleFilterError
 
+DOCUMENTATION = r"""
+---
+name: snmp_hash
+collection: arista.avd
+author: Arista Ansible Team (@aristanetworks)
+version_added: "3.6.0"
+short_description: Compute localized SNMP passphrases
+description:
+  - Key localization as described in L(RFC 2574 section 2.6,https://www.rfc-editor.org/rfc/rfc2574.html#section-2.6)
+positional: _input
+options:
+  _input:
+    description: Dictionary with snmp passphrase details.
+    type: dictionary
+    required: true
+    suboptions:
+      passphrase:
+        type: string
+        required: true
+        description: The passphrase to localize, if I(priv) is not set it is the "auth" passphrase else it is the "priv" passphrase.
```

Comment:
Hard to read sentence. Consider rewriting

Reply:
better?